stable/nginx-ingress/values.yaml

```diff
@@ -8,6 +8,11 @@ controller:
     tag: "0.19.0"
     pullPolicy: IfNotPresent
 
+  configMap:
+    ## point to an existing configMap rather than creating one
+    name: ""
```

Comment:
use the null character `~` rather than an empty string.  Also I don't think there's a need for it to be a nested value.

`controller.configMap: ~` would work and you'll be able to use `{{ if ! .Values.controller.configMap }}` to test for it.